app/src/pages/trace/TracePage.tsx

```diff
@@ -214,7 +214,10 @@ export function TracePage() {
         }
       }
     `,
-    { traceId: traceId as string }
+    { traceId: traceId as string },
+    {
+      fetchPolicy: "network-only",
+    }
```

Comment:
Do you still need this? Alternatively you can use fetch-and-network https://relay.dev/docs/guided-tour/reusing-cached-data/fetch-policies/